pennylane_lightning/core/src/measurements/MeasurementsBase.hpp

```diff
@@ -17,13 +17,37 @@
  */
 #pragma once
 
+#include <regex>
+#include <string>
 #include <vector>
 
 #include "Observables.hpp"
 
 /// @cond DEV
 namespace {
 using namespace Pennylane::Observables;
+void parse_obs2ops(const std::string &obs_name, std::vector<std::string> &ops,
```

Comment:
this can be removed (just adding for the future reference)

Reply:
Removed.